libs/entities/src/entities/governance.config.ts

```diff
@@ -0,0 +1,33 @@
+import { BinaryUtils } from '@multiversx/sdk-nestjs-common';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class GovernanceConfig {
+  constructor(init?: Partial<GovernanceConfig>) {
+    Object.assign(this, init);
+  }
+
+  @ApiProperty({ description: 'The fee required for submitting a new proposal.' })
+  proposalFee?: string;
+
+  @ApiProperty({ description: 'The minimum quorum required for a proposal.' })
+  minQuorum?: number;
+
+  @ApiProperty({ description: 'The minimum pass threshold required for a proposal.' })
+  minPassThreshold?: number;
+
+  @ApiProperty({ description: 'The minimum veto threshold required for a proposal' })
+  minVetoThreshold?: number;
+
+  @ApiProperty({ description: 'The nonce of the last proposal.' })
+  lastProposalNonce?: number;
+
+  static fromVmQueryResponse(queryResponse: any): GovernanceConfig {
+    return new GovernanceConfig({
```

Comment:
also you can destruct the query response and be able to use like this for example:
`const returnData = queryResponse?.data?.data?.returnData;`

Reply:
refactored